forms-flow-web/src/components/Dashboard/Dashboard.js

```diff
@@ -166,6 +172,27 @@ const Dashboard = React.memo(() => {
     dispatch(setMetricsDateRangeLoading(true));
     dispatch(setMetricsDateChange(date));
   };
+  const headerList = () => {
+    return [
+      {
+        name: "Metrics",
+        count: totalItems,
+        onClick: () => dispatch(push(`${redirectUrl}metrics`)),
+        icon: "pie-chart",
+      },
+      {
+        name: "Insights",
+        onClick: () => dispatch(push(`${redirectUrl}insights`)),
+        icon: "lightbulb-o",
+      },
+    ];
+  };
+
+  let headOptions = headerList();
+
+  if (!getUserInsightsPermission) {
```

Comment:
We cam remove this condition. Had a discussion with @sreehari-aot

Reply:
condition removed